app/services/docker/containerService.js

```diff
@@ -29,7 +29,10 @@ angular.module('portainer.services')
       deferred.resolve(containers);
     })
     .catch(function error(err) {
-      deferred.reject({ msg: 'Unable to retrieve containers', err: err });
+      throw {
```

Comment:
Why use a throw instead of the `reject()` method of the deferred object?

Reply:
Missed that on merge, I'll replace it with the deferred object.